moped-database/metadata/tables.yaml

```diff
@@ -3324,249 +3316,224 @@
       permission:
         check: {}
         columns:
-          - added_by
           - capitally_funded
-          - current_phase
-          - current_phase_id
-          - current_status
-          - date_added
-          - ecapris_subproject_id
           - end_date
-          - fiscal_year
-          - is_deleted
-          - knack_project_id
+          - added_by
+          - interim_project_id
           - milestone_id
-          - project_description
-          - project_description_public
+          - parent_project_id
           - project_id
           - project_importance
           - project_lead_id
           - project_length
-          - project_name
           - project_order
-          - project_priority
           - project_sponsor
-          - project_uuid
-          - project_website
-          - status_id
-          - task_order
           - timeline_id
-          - updated_at
+          - task_order
+          - ecapris_subproject_id
+          - fiscal_year
+          - knack_project_id
+          - project_description
+          - project_description_public
+          - project_name
+          - project_priority
+          - project_website
           - work_assignment_id
-          - parent_project_id
-          - interim_project_id
+          - date_added
+          - updated_at
+          - project_uuid
+          - is_deleted
```

Comment:
on `moped_project`, we're dropping:
- current_phase
- current_phase_id
- current_status
- status_id